tempesta_fw/t/unit/test_sched_hash.c

```diff
@@ -189,6 +192,8 @@ TEST(tfw_sched_hash, max_srv_in_sg_and_max_conn)
 
 TEST_SUITE(sched_hash)
 {
+	tfw_http_parse_req_ptr = get_sym_ptr("tfw_http_parse_req");
+
```

Comment:
A `BUG_ON(tfw_http_parse_req_ptr == NULL);` here will help to catch `NULL` as early as possible.